### DISCUSSION TITLE
missing unzip and npm leads to LSP error

### DISCUSSION THREAD

**Discussion by fabianmuehlberger** [LazyVim/LazyVim#1342] [General]
I noticed on a new install that my MasonLog show:
``` 
[ERROR Thu 17 Aug 2023 12:34:05 PM UTC] ...e/nvim/lazy/mason.nvim/lua/mason-core/installer/init.lua:249: Installation failed for Package(name=json-lsp) error=spawn: npm failed with exit code - and signal -. npm is not executable
[ERROR Thu 17 Aug 2023 12:34:06 PM UTC] ...e/nvim/lazy/mason.nvim/lua/mason-core/installer/init.lua:249: Installation failed for Package(name=stylua) error=spawn: unzip failed with exit code - and signal -. unzip is not executable
```
Due to LSP being activated at install, this might lead to confusion for new users. 

I suggest adding  (LSP/unzip) as a requirement or dependency to the getting started or the plugin/LSP page.

Best Fabian

**Comment by dpetka2001** [comment]
This is not a LSP issue. It's a `mason` issue and how it fetches and distributes the packages. I don't believe it's Lazyvim's responsibility to see to this. It could be mentioned on the Lazyvim's website under `mason` section, that this package depends on these tools to fetch and install packages, however there shouldn't be a need to make these tools requirements of Lazyvim's codebase.

**Reply by fabianmuehlberger** [reply]
That seems helpful. Not long ago, I had trouble finding solutions to seemingly small problems, this is why I mentioned it.
